roles/cifmw_cephadm/tasks/install_cephadm.yml

```diff
@@ -30,6 +30,10 @@
   ansible.builtin.dnf:
     name: cephadm
     state: present
+  register: task_result
+  retries: 8
```

Comment:
maybe not important, just out of curiosity: how did we come up w/ these numbers? is there any best practice to set defaults? Would it be good to define two variables under `roles/cifmw_cephadm/default/main.yaml` with those numbers, so we can eventually pass them as overrides to the jobs?
Apart from this small potential change, I'm not against this patch and we can land it.

Reply:
Thanks, good point. I've just added the default values.